eclipsesource/cdtcloud-deploymentserver
# feat(theia-extension): add configurability of host in the extension s…

…ettings

## Changes Proposed:


- add preferences for the theia-extension to change host

## Issues Addressed:


- Closes #85 

## Tests Performed:


- run application, change host while running


## How to Test the Changes:


1. run the application
2. Preferences: Open Workspace Settings
3. under extensions change Deployment Server Host or Deployment Server Secure
4. if there is no server running on the new host there should be errors.
5. change it back and the deployment to a device should work again

## Known Issues and TODO List:


## How to Test PRs
1) First fetch the Pull-Request from the main repo, replacing `xx` with the ID of the PR:\
`git fetch git@github.com:eclipsesource/cdtcloud-deploymentserver.git pull/xx/head:pr_xx`
2) Checkout the Pull-Request, again replacing `xx` with the PR-ID:\
`git checkout pr_xx`
3) Perform the necessary tests for the PR
